### PR TITLE
Add missing conversation programs to campus dictionary

### PR DESCRIPTION
**Pull request by zimmer18**
Added 
- Asian Conversations
- Environmental Conversation
- Public Affairs Conversation
- Science Conversation

to the campus dictionary and changed abbreviation of Am Con to match all other abbreviations.

### CODE REVIEW

File: data/dictionary/asian-conversations.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+word: Asian Conversations
```
Comment (hawkrives) [inline data/dictionary/asian-conversations.yaml]: Shouldn't this be Asian Con, to match the others?

File: data/dictionary/amcon.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-word: Amcon
+word: Am Con
```
Comment (hawkrives) [inline data/dictionary/amcon.yaml]: Capitalize the Con part to match the others?

**Review by hawkrives — CHANGES_REQUESTED** [review]
Two tweaks maybe?

File: data/dictionary/asian-conversations.yaml (hunk 1 above)
Comment (zimmer18) [reply data/dictionary/asian-conversations.yaml]: Yup!

File: data/dictionary/amcon.yaml (hunk 2 above)
Comment (hawkrives) [reply data/dictionary/amcon.yaml]: Was this already capitalized and I misread it? Looks like you can ignore this comment.

Comment (hawkrives) [reply data/dictionary/amcon.yaml]: Please change this to "AmCon" as we discussed separately.

File: data/dictionary/amcon.yaml [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 definition: >
   Short for American Conversations, an interdisciplinary General Education
   program and learning community in which students analyze America’s history
   and culture. Over four successive semesters in their first and sophomore
-  years, Amcon students encounter many of the texts, topics, and historical
+  years, Am Con students encounter many of the texts, topics, and historical
```
Comment (hawkrives) [inline data/dictionary/amcon.yaml]: ... and fix it in here too, plz?

File: data/dictionary/asian-conversations.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+word: Asian Con
```
Comment (hawkrives) [inline data/dictionary/asian-conversations.yaml]: s/Asian Con/AsianCon/

File: data/dictionary/encon.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+word: En Con
```
Comment (hawkrives) [inline data/dictionary/encon.yaml]: Um. EnCon, unless they followed in the footsteps of Great Con?

You might need to find someone and ask them.

File: data/dictionary/paccon.yaml [hunk 6]
```diff
@@ -0,0 +1,18 @@
+word: Pac Con
```
Comment (hawkrives) [inline data/dictionary/paccon.yaml]: Find someone on-campus and ask? PaCon/PacCon/Pac Con/I dunno

File: data/dictionary/scicon.yaml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+word: Sci Con
```
Comment (hawkrives) [inline data/dictionary/scicon.yaml]: SciCon

**Review by hawkrives — CHANGES_REQUESTED** [review]
As @zimmer18 and I discussed in Slack, we're going to switch these to the on-campus colloquial versions.

**Review by hawkrives — APPROVED** [review]
Thanks!

**Pull request merged**